tests/fsharp/typecheck/sigs/neg117.bsl

```diff
@@ -1,5 +1,5 @@
 
-neg117.fs(79,18,79,59): ilxgen error FS0041: No overloads match for method 'Transform'.
+neg117.fs(74,51,74,121): ilxgen error FS0041: No overloads match for method 'Transform'.
```

Comment:
This change in error location is acceptable. This is a late last-minute check from IlxGen based on inline code which now reports a different range.  The error should really be reported during type checking, or the code generation succeed.